### PR TITLE
ENH: Speed up a couple of tests

### PR DESCRIPTION
**Pull request by larsoner**
Just speeding up a few tests that were listed as slow in #5251

### CODE REVIEW

**Comment by larsoner** [comment]
With these changes testing time is lower to the point I think we can combine runs (i.e., remove the `SPLIT` business). The setup for conda takes over 5 min so we will ultimately save time doing this. Okay with you @agramfort ?

**Comment by larsoner** [comment]
This makes things quite a bit faster so I'm going to merge, but feel free to look if you have time later and I can adjust

File: mne/io/bti/bti.py [hunk 1]
```diff
@@ -1016,7 +1016,7 @@ class RawBTi(BaseRaw):
         and :ref:`Logging documentation <tut_logging>` for more).
     """
 
-    @verbose
+    # @verbose
```
Comment (agramfort) [inline mne/io/bti/bti.py]: undo?

Comment (larsoner) [reply mne/io/bti/bti.py]: argh yes :(

**Pull request merged**